### ISSUE TITLE
Add support to @nanoid

### ISSUE THREAD

**Issue by dikyarga** [zenstackhq/zenstack#923]
**Description and expected behavior**

I got error when using `@nanoid`, `@nanoid` is supported by Prisma since v4.9.0 

sample code:

```
model Brand {
  id          String      @id @default(nanoid(3))
}
```

**Screenshots**

![CleanShot 2024-01-03 at 05 07 13@2x](https://github.com/zenstackhq/zenstack/assets/6753674/532b4bec-ed24-4c76-8eba-0dde41a95c78)

**Environment (please complete the following information):**

-   ZenStack version: [1.6.0]
-   Prisma version: [5.3.1]
-   Database type: [SQLite]

**Additional context**

Link to Prisma issue: https://github.com/prisma/prisma/issues/14004

**Comment by ymc9** [comment]
Thanks for filing this @dikyarga . Let's get it fixed soon. Let me know if you're interested in making a PR 😄. All attribute functions are defined here: https://github.com/zenstackhq/zenstack/blob/bdb95ff4d8181fb63d342b9dc434c91eed87abe9/packages/schema/src/res/stdlib.zmodel

**Comment by dikyarga** [comment]
yes yes @ymc9, I'm interested in working on this, thanks for the reference and excellent toolkit!

**Comment by ymc9** [comment]
Great. Thank you!

You can fork and make a PR off the "dev" branch. I'll be happy to test it and offer help as needed.

**Comment by dikyarga** [comment]
PR raised @ymc9, I tried to add a test and run it, but I am not sure if it is enough or not.

Thank you so much for your help!

**Comment by ymc9** [comment]
The fix is live in v1.6.1 now. Please upgrade both the npm packages and VSCode extension. Thanks for working on this fix! @dikyarga
